AbbaGarima/AG00001.xml

```diff
@@ -994,7 +994,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="NV" when="2019-10-21">Created an entry for Sergey Kim</change>
+            <change who="NV" when="2019-10-21">Created a stub for Sergey Kim</change>
```

Comment:
I don't understand which change in this xml record youare describing. Note our rule for the use of word stub